app/src/test/java/com/codesoom/assignment/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,123 @@
+package com.codesoom.assignment.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.BDDMockito.given;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.errors.UserAuthenticationFailedException;
+import com.codesoom.assignment.errors.UserEmailNotExistException;
+import com.codesoom.assignment.utils.JwtUtil;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+@DisplayName("AuthenticationService 클래스")
+class AuthenticationServiceTest {
+  final Long USER_ID = 1L;
+  final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.5VoLz2Ug0E6jQK_anP6RND1YHpzlBdxsR2ORgYef_aQ";
+  final String secretKey = "qwertyuiopqwertyuiopqwertyuiopqw";
+  final String VALID_EMAIL = "rlawlstjd@gmail.com";
+  final String VALID_PASSWORD = "12345";
+
+  final User GIVEN_USER = User.builder()
+      .id(1L)
+      .name("rlawlstjd")
+      .email(VALID_EMAIL)
+      .password(VALID_PASSWORD)
+      .build();
+
+  AuthenticationService authService;
+
+  @BeforeEach
+  void setUp() {
+    UserRepository userRepository = Mockito.mock(UserRepository.class);
+    authService = new AuthenticationService(new JwtUtil(secretKey), userRepository);
+
+    given(userRepository.findByEmail(VALID_EMAIL)).willReturn(Optional.of(GIVEN_USER));
+  }
+
+  @Nested
+  @DisplayName("encode()")
```

Comment:
굳이 호출 연산자를 같이 적어주지 않아도 괜찮을 것 같아요.
```suggestion
  @DisplayName("encode")
```

Reply:
메서드 라는 의미를 알려주고자 했는데 없어도 괜찮겠군요